### PR TITLE
FTP.DownloadFiles - Modified operations log

### PR DESCRIPTION
**Pull request by RikuVirtanen**
#19

### CODE REVIEW

**Comment by RikuVirtanen** [comment]
Detect-Errors fail because there are recommendations that are not fixed. Those should not be fixed because the error handling is done specifically for catching any errors coming from the code.

File: Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles.Tests/Lib/FtpHelper.cs [hunk 1]
```diff
@@ -118,4 +118,13 @@ public void DeleteDirectoryOnFTP(string ftpDir)
     {
```
Comment (Svenskapojkarna) [inline Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles.Tests/Lib/FtpHelper.cs:117]: https://github.com/FrendsPlatform/Frends.FTP/blob/463232756c958a4b5205078555a1ebed3672ec6c/Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles.Tests/Lib/FtpHelper.cs#L27
Please remove commented line.

File: Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles.Tests/Lib/FtpHelper.cs [hunk 2]
```diff
@@ -118,4 +118,13 @@ public void DeleteDirectoryOnFTP(string ftpDir)
     {
         client.DeleteDirectory(ftpDir, FtpListOption.Recursive);
     }
+
+    public void RemoveLocalTestFiles()
```
Comment (Svenskapojkarna) [inline Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles.Tests/Lib/FtpHelper.cs]: This method can be static.

File: Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles.Tests/MacrosTests.cs [hunk 3]
```diff
@@ -1,109 +1,109 @@
...
-        string sourceDirectory,
-        string sourceFileName,
-        string targetDirectory,
-        string targetFileName = null,
-        string moveToDir = null,
-        string renameTo = null)
-    {
-        var source = new Source
-        {
+    private Result CallDownloadFiles(
```
Comment (Svenskapojkarna) [inline Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles.Tests/MacrosTests.cs]: This method can be static.

**Review by Svenskapojkarna — COMMENTED** [review]
Please also increase Task version.

**Review by Svenskapojkarna — APPROVED** [review]

**Pull request merged**